react/week-3/lesson.md

```diff
@@ -42,41 +46,57 @@ class Counter extends Component {
 
 ## Unmounting
 
-So far we've looked at components that are always rendered in the browser. However (and this is often the case in large applications), we might want to control whether components are shown or not. Let's look at a Toggle component ([interactive example](https://codesandbox.io/s/xmo8oo514)):
+So far we've looked at components that are always rendered in the browser. However (and this is often the case in large applications), we might want to control whether components are shown or not. 
```

Comment:
Everything up to the forms section in week 3 is stuff I've been working on. Perhaps we could move these changes/discussion to that PR when I open it (Soon™️)?

Reply:
Yes sorry I got carried away 😬